src/themes/eui-amsterdam/overrides/_flyout.scss

```
@@ -2,4 +2,4 @@
 // The clip path ensures only the left side of the shadow is exposed.
 .euiFlyout {
   clip-path: polygon(-10% 0, 100% 0, 100% 100%, -10% 100%);
-}
+}
```

nit: Space at the end of doc
```suggestion
}

```